package.json

```diff
@@ -14,13 +14,19 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@google-cloud/pubsub": "^0.30.2",
-    "@types/node": "^10.11.4",
-    "@types/request": "^2.47.1",
-    "@types/request-promise": "^4.1.42",
-    "request": "^2.87.0",
-    "request-promise": "^4.2.2",
-    "tslint-config-ackee": "^0.1.1",
-    "typescript": "^3.1.6"
+    "@google-cloud/pubsub": "^1.6.0",
+    "@types/amqplib": "^0.5.13",
```

Comment:
move it to the `devDependencies` :pray:

Reply:
Those are non-dev. We are exposing them via adapters.